symplyphysics/laws/thermodynamics/diffusion_coefficient_of_spherical_brownian_particles_from_temperature_and_dynamic_viscosity.py

```diff
@@ -0,0 +1,43 @@
+from sympy import (Eq, solve, pi)
+from symplyphysics import (units, Quantity, Symbol, print_expression, validate_input,
+    validate_output)
+
+# Description
+## Brownian motion is the random motion of microscopic visible suspended particles of a solid substance in a liquid or gas caused by the thermal motion of particles of a liquid or gas.
+## The diffusion coefficient is a quantitative characteristic of the diffusion rate, equal to the amount of matter passing per unit time through a section of a unit area as a result of the thermal motion of molecules with a concentration gradient equal to one.
+## The diffusion coefficient is determined by the properties of the medium and the type of diffusing particles.
+
+## Law: D = R * T / (6 * Na * pi * r * eta)
+## Where:
+## D is diffusion coefficient
+## R is universal gas constant
+## T is temperature
+## Na is avogadro number
+## r is particle radius
+## eta is dynamic viscosity
+
+## Conditions
+## Particle displacements in any direction are equally likely
+## The inertia of a Brownian particle can be neglected compared to the influence of friction forces
+
+diffusion_coefficient = Symbol("diffusion_coefficient", units.area / units.time)
```

Comment:
Might be a good idea to add `units.amount_of_substance` to indicate that it is amount of substance through some area per second.